tests/test_app.py

```diff
@@ -18,6 +19,10 @@ def test_app_binds(self):
         self.app.bind('test2', object)
         assert self.app.providers == {'test1': object, 'test2': object}
 
+    def test_app_simple_bind(self):
```

Comment:
You should also try to bind an instantiated object as well. I have a sneaky suspicion that it will throw an exception because an instantiated object has no attribute `__name__`

Reply:
you might have to do someting like `object.__name__ or object.__class__.__name__` or something

Reply:
might look something like:

```python
        self.bind("{module}.{name}".format(
            module=obj.__module__, name=obj.__name__ or obj.__class__.__name__ ), obj)
```

maybe?